projects/module-2/fitlit-part-two-agile.md

```diff
@@ -218,44 +218,63 @@ Once your usability test is complete, incorporate any useful and interesting fee
 
 ## Project Requirements Rubric
 
-### Functional Expectations
+For the rubric sections below, you will be scored as **Wow**, **Yes** or **Not Yet** depending on whether you have demonstrated competency in that area. Each section lists examples of what types of things we may be looking for as demonstrations of competency. Just as there are many ways to approach code, there are many many ways to demonstate competency.  These are just *some* examples. 
 
-- 4: Application fulfills all iteration requirements, without bugs, and the user experience is intuitive and consistent.
-- 3: Application fulfills all iteration requirements, including implementing feedback from Part 1 and feedback from the usability test.
-- 2: Application is usable but has some missing functionality.
-- 1: Application crashes during normal usage or does not run.
+<section class="answer">
+### Does the project demonstrate student understanding of the learning goals & concepts?
 
-### Testing
+Projects will answer that question, being marked as yes, not yet, and wow. Similarly, each section of the rubric (see below) will have yes/not yet/wow markings, helping you understand your progress and growth in specific areas.
+
+The overall project outcome (yes, not yet, wow) is determined by “averaging” each section’s outcome. You can think of a “yes” being worth a 1, a “not yet” being worth a 0, and a “wow” being worth a 2.
+
+For this project, an average of 0.5 is considered a yes - a passing project that demonstrates good student understanding! An average of 1+ is considered a wow. Anything below a 0.5 is considered a not yet - a project that indicates that the concepts have not been fully understood (see note in the section below). 
+</section>
+
+While M2 rubrics do not have a separate section for WOWs like in M1, there are a few WOW examples noted throughout.  In addition to these WOW bullets, you can strive for a WOW by demonstrating not just competency, but excellence and thoroughness across the rubric sections. 
```

Comment:
Love this extra little note to give clarity when there are parts of the rubric that don't have a WOW.  I think pushing students striving to master the core concepts and being thorough is just as important, if not MORE important, than doing another thing.  Cheers!